Lib/lib2to3/tests/test_parser.py

```diff
@@ -84,12 +84,16 @@ def test_load_grammar_from_subprocess(self):
             # different hash randomization seed.
             sub_env = dict(os.environ)
             sub_env['PYTHONHASHSEED'] = 'random'
-            subprocess.check_call(
-                    [sys.executable, '-c', """
+            code = """
 from lib2to3.pgen2 import driver as pgen2_driver
 pgen2_driver.load_grammar(%r, save=True, force=True)
-                    """ % (grammar_sub_copy,)],
-                    env=sub_env)
+            """ % (grammar_sub_copy,)
+            msg = ("lib2to3 package is deprecated and may not be able "
+                   "to parse Python 3.10+")
```

Comment:
In PR #25406, I added distutils._DEPRECATION_MESSAGE constant to avoid copying the error message. Is it worth it?